daemon/openrazer_daemon/hardware/keyboards.py

```diff
@@ -984,11 +983,11 @@ class RazerBlade2018Mercury(_RippleKeyboard):
 
     USB_VID = 0x1532
     USB_PID = 0x0240
-    HAS_MATRIX = True
+    HAS_MATRIX = False
     MATRIX_DIMS = [6, 16]
     METHODS = ['get_device_type_keyboard', 'set_wave_effect', 'set_static_effect', 'set_spectrum_effect',
                'set_reactive_effect', 'set_none_effect', 'set_breath_random_effect', 'set_breath_single_effect',
-               'set_breath_dual_effect', 'set_custom_effect', 'set_key_row', 'set_starlight_random_effect',
+               'set_breath_dual_effect', 'set_custom_effect', 'set_starlight_random_effect',
```

Comment:
this device wasn't mentioned in your test results and it seems to have everything in the driver?

Reply:
Oops, you're right. My bad, I must've got tangled up. Just undid this change, tested and there's no fault with this one.